### ISSUE TITLE
nmdc-schema should support generation of DH interfaces from standardized terms in current standards like MIxS 6

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#283]
This has been described as "the schema should no include any copied instances from MIxS 5". I think that means that we should no longer import [src/schema/mixs.yaml](https://github.com/microbiomedata/nmdc-schema/blob/main/src/schema/mixs.yaml) into [src/schema/nmdc.yaml](https://github.com/microbiomedata/nmdc-schema/blob/main/src/schema/nmdc.yaml). I still don't understand the mechanism that generated that file, but it may be moot now.

## Challenges

1. There is no web-resolvable route to a MIxS 6 LinkML artifact yet
  1. w3id resolution to the file artifacts and the per-tem docs pages has not been implemented yet
  2. ~~the MIxS 6 LinkML is not even part of the [main GSC MIxS branch](https://github.com/GenomicsStandardsConsortium/mixs) yet~~
  3. minor: the gen-docs per-term pages need improvements to their content and aesthetics
2. ["it comes from MIxS" assertions](https://github.com/microbiomedata/nmdc-server/issues/555) have been requested for the MIxS terms when they appear in nmdc-schema artifacts.
  - ~~intend to implement with subset slots~~
  - implemented with `source` slot, with a value that comes from the `from_schema` slot
  - does this require some kind of overlay file? we can't/shouldn't put it in the imported MIxS 6 files can we?

CC @cmungall @mslarae13 @sujaypatil96

**Comment by turbomam** [comment]
what is the driver for having MIxS slots annotated as "I came from MIxS"? How would that be used in the portal?

Might be available in from_slot anyway (and may not need new subset)

import the entire new huge MIxS 6? even for slots we might never use?

old NMDC mixs.ymal ~ 500 terms, slots only

those slots are manually added to the biosample class definition in XXX.yaml

ranges overridden with complex types/classes

**Comment by turbomam** [comment]
should sheets and friends get mixs knowledge from nmdc for DH, or should sheets and friends modify mixs for inclusion in nmdc schema itself?!

**Comment by ssarrafan** [comment]
Moving this to April per @turbomam 
@dehays @cmungall @emileyfadrosh can you respond to Mark's last comment above?

**Comment by emileyfadrosh** [comment]
Sorry, I am not following the nuances here -- I think @cmungall needs to weigh in. I think from my naive perspective, I would like to have the NMDC schema & DH templates aligned with MIxS v6, yet also include curation updates from @mslarae13 where there are updates needed that haven't been added to v6 yet. We discussed this briefly how NMDC would serve as an intermediate between the current MIxS version and the future version. Not sure if this is helpful, sorry!

@cmungall input please!

**Comment by turbomam** [comment]
After some reflection and discussion, I would say that this issue is a little vague and may even combine tasks that are more schema related or are more DH-related. But it's a good issue because it emphasizes that the overall NMDC schema and the DH-specific schema shouldn't conflict with one another, and if they do, it should be documented in the schams.  

Also, as the issue title says, none of this is really specific to MIxS. The overall NMDC schema and the DH-specific schema should be able to inform on another about proper usage of terms from any external schema or ontology.

See progress in [Indicate MIxS terms #555]( https://github.com/microbiomedata/nmdc-server/issues/555)

I have found workarounds for most of the challenges I listed above.
